src/ceph-volume/ceph_volume/devices/raw/prepare.py

```diff
@@ -132,5 +172,8 @@ def main(self):
         if not self.args.bluestore:
             terminal.error('must specify --bluestore (currently the only supported backend)')
             raise SystemExit(1)
+        if self.args.dmcrypt and not os.getenv('CEPH_VOLUME_DMCRYPT_SECRET'):
+            terminal.error('Could not read value from environment variable CEPH_VOLUME_DMCRYPT_SECRET, you must set this variable.')
```

Comment:
```suggestion
            terminal.error('Could not read value from environment variable CEPH_VOLUME_DMCRYPT_SECRET, you must set this variable when passing --dmcrypt.')
```

Maybe lets print a bit more explicit error message?